net/httpclient.go

```diff
@@ -80,6 +84,10 @@ func NewTransport(options Options) *Transport {
 		options.Tracer = &opentracing.NoopTracer{}
 	}
 
+	if options.Timeout == 0 {
+		options.Timeout = defaultTimeout
```

Comment:
let's only enforce the IdleConnTimeout. The other settings have sane defaults.

Reply:
ok

Reply:
fixed